packages/terra-pills/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "terra-pills",
+  "main": "lib/index.js",
```

Comment:
Make the package private until first release.
```suggestion
  "main": "lib/index.js",
  "private": true,
```

Reply:
changed here 5158250e39187534d43a615c7ee12bd447c7ab6b